cmd/podman/artifact/list.go

```diff
@@ -38,12 +38,29 @@ type listFlagType struct {
 }
 
 type artifactListOutput struct {
-	Digest      string
-	Repository  string
-	Size        string
-	Tag         string
-	Created     string
-	VirtualSize string
+	Digest       string
+	Repository   string
+	Size         string
+	Tag          string
+	created      time.Time
+	VirtualSize  string
+	virtualBytes int64
+}
+
+// Created returns human-readable elapsed time since artifact was created
+func (a artifactListOutput) Created() string {
```

Comment:
Should this be `CreatedHuman`? That would match `podman ps`. `Created` should be an alias for `CreatedAt`